xmlPivotTable.go

```diff
@@ -31,7 +31,7 @@ type xlsxPivotTableDefinition struct {
 	DataCaption             string                   `xml:"dataCaption,attr"`
 	GrandTotalCaption       string                   `xml:"grandTotalCaption,attr,omitempty"`
 	ErrorCaption            string                   `xml:"errorCaption,attr,omitempty"`
-	ShowError               bool                     `xml:"showError,attr,omitempty"`
+	ShowError               *bool                     `xml:"showError,attr,omitempty"`
```

Comment:
If the specified pointer of boolean type for this field, the `omitempty` JSON struct tag can be removed, we can also remove it in `ShowDrill` JSON tags.

Reply:
Thanks for your review and suggestion. I've applied what you mentioned above. 
- gofmt
- remove json omitempty tag for both ShowError, ShowDrill